src/core/models/searchconfig.js

```diff
@@ -11,6 +11,12 @@ export default class SearchConfig {
      */
     this.limit = config.limit || 20;
```

Comment:
You may want to consider renaming `limit` here. I could see someone getting confused about how `limit` vs `universalLimit` works. They might think `universalLimit` allows us to specify limits per vertical, but `limit` is cumulative over all verticals. 

One way to avoid this confusion could be renaming `limit` to something like `limitForVertical`. Adding more info to the JSDocs about when one is used versus the other could also help disambiguate.

Reply:
Tom, you are suggesting to rename the internal SearchConfig property and not the public config option, correct? Otherwise that would be a breaking change